### PR TITLE
Arm64 support as documented here: …

### PR DESCRIPTION
**Pull request by ArielleA**
http://zhiyisun.github.io/2017/02/15/Running-Google-Machine-Learning-Library-Tensorflow-On-ARM-64-bit-Platform.html

These instructions are a little out of date so I have re-biased the patch. 
Not included in this patch are changes to the iOS and Android rulebases.

### CODE REVIEW

**Comment by ArielleA** [comment]
I signed it

**Comment by ArielleA** [comment]
My main workstation is a Samsung Chromebook Plus running Debian Stretch, which is arm64 architecture. The patch is allow me to build my project locally using Bazel.
Also as Tensorflow is quite widely used, adding support for Arm64 would be useful especially as SOCs with accelerators are becoming available. 
There is a related issue is bazelbuild/rules_go#1506

**Comment by laszlocsomor** [comment]
@ulfjack: ping

**Comment by futurejones** [comment]
I have been testing Bazel on aarch64 systems with these changes. 
https://github.com/futurejones/bazel/tree/futurejones-aarch64-patch
(Obtained from the same source as above.)
http://zhiyisun.github.io/2017/02/15/Running-Google-Machine-Learning-Library-Tensorflow-On-ARM-64-bit-Platform.html
I can confirm successful builds and use of Bazel on a variety of arm64/aarch64 machines.

File: tools/platforms/platforms.BUILD [hunk 1]
```diff
@@ -4,6 +4,18 @@ package(
     default_visibility = ["//visibility:public"],
 )
 
+filegroup(
```
Comment (ulfjack) [inline tools/platforms/platforms.BUILD]: What is this needed for?

**Comment by ArielleA** [comment]
Sorry, that was my error, those lines should not have been there.

**Comment by ulfjack** [comment]
@dslomov

File: tools/platforms/BUILD [hunk 2]
```diff
@@ -107,6 +112,7 @@ platform(
         ":x86_64",
         ":ppc",
         ":arm",
+	":aarch64",
```
Comment (steeve) [inline tools/platforms/BUILD]: indentation is slightly wrong

File: tools/platforms/BUILD [hunk 3]
```diff
@@ -128,6 +134,7 @@ platform(
...
```
Comment (steeve) [inline tools/platforms/BUILD]: indentation is slightly wrong

File: tools/platforms/platforms.BUILD [hunk 4]
```diff
@@ -95,6 +100,7 @@ platform(
         ":x86_64",
         ":ppc",
         ":arm",
+	":aarch64",
```
Comment (steeve) [inline tools/platforms/platforms.BUILD]: indentation is slightly wrong

File: tools/platforms/platforms.BUILD [hunk 5]
```diff
@@ -116,6 +122,7 @@ platform(
...
```
Comment (steeve) [inline tools/platforms/platforms.BUILD]: indentation is slightly wrong

**Comment by ArielleA** [comment]
Indentation fixed thankyou :)

**Pull request closed without merging**